### DISCUSSION TITLE
When writing to Parquet, what is the default row group size?

### DISCUSSION THREAD

**Discussion by rmx90210** [duckdb/duckdb#13809] [Q&A] [answered]
I could find this in the [documentation](https://duckdb.org/docs/data/parquet/overview). I can PR an edit to add it, if anyone knows the answer. Thanks!

**Comment by soerenwolfers** [comment, marked as answer]
https://duckdb.org/docs/data/parquet/tips.html#:~:text=The%20ROW_GROUP_SIZE%20parameter%20specifies%20the,each%20column%20in%20the%20dataset.
